lib/msal-core/src/Utils.ts

```diff
@@ -225,136 +223,29 @@ export class Utils {
 
   //#region Encode and Decode
 
+  // See: https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding#Solution_4_%E2%80%93_escaping_the_string_before_encoding_it
+
   /**
    * encoding string to base64 - platform specific check
    *
    * @param input
    */
-  static base64EncodeStringUrlSafe(input: string): string {
-    // html5 should support atob function for decoding
-    return Base64.encode(input);
+  static base64Encode(input: string): string {
+    return btoa(encodeURIComponent(input).replace(/%([0-9A-F]{2})/g,
+        function toSolidBytes(match, p1) {
+            return String.fromCharCode(Number('0x' + p1));
```

Comment:
This build is failing for me locally because of the single quoted strings here and below.

ERROR: src/Utils.ts:236:47 - ' should be "
ERROR: src/Utils.ts:246:49 - ' should be "
ERROR: src/Utils.ts:247:16 - ' should be "
ERROR: src/Utils.ts:247:23 - ' should be "
ERROR: src/Utils.ts:248:13 - ' should be "

I'm not sure how travis is passing, but might want to double check your local build.